projects/pastanaga-angular/src/styles/_layout.scss

```diff
@@ -52,23 +52,36 @@ $footer-height: rhythm(6) * 3;
     .pa-main-side-panel {
         width: $width-side-panel;
     }
+    .pa-main-dock {
+        width: $width-dock;
+    }
 
     .pa-main-container {
         display: flex;
         width: 100%;
     }
+
     .pa-main-side-panel + .pa-main-container-wide,
     .pa-main-container-wide.pa-side-on-right {
         width: $width-formula-panel-plus-wide-medium;
         margin: 0 auto;
     }
+
```

Comment:
We could add this after line 65 to avoid duplication or there is some difference between both?